### DISCUSSION TITLE
Bold text of navbar items

### DISCUSSION THREAD

**Discussion by shivarm** [Hyperfoil/Horreum#485] [Announcements]
**Feature idea.**
navbar items not looking clear and it needs to be clear to user

**Comment by karthiknadar1204** [comment]
I would like to work on this issue,please assign it to me

**Comment by jesperpedersen** [comment]
This is not an issue yet - moved to the forum.

One of you - create a prototype with the changes, and show screenshots of before and after.

Thanks !

**Reply by shivarm** [reply]
okay @jesperpedersen

**Comment by shivarm** [comment]
change: Text style of navbar items #487
